tests/ios/frameworks/objc/BUILD.bazel

```diff
@@ -9,6 +9,10 @@ apple_framework(
         "main.m",
     ],
     platforms = {"ios": "10.0"},
+    resource_bundles = {"FooResources": glob(
+        ["Resources/**/*"],
```

Comment:
can we add another Resources dir (Resources2) to verify that both dirs get cloned over?

Reply:
Added